apps/core/components/ItineraryCard/itineraryDetail/SectorsList.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import type { SectorsList as SectorType } from './__generated__/SectorsList.graphql';
+import SectorDetail from '../../sectorDetail/SectorDetail';
+
+type Props = {|
+  +data: ?SectorType,
+|};
+
+function SectorsList(props: Props) {
+  const sectors = props.data?.sectors ?? [];
+  return sectors.map((sector, sectorIndex) => {
+    return <SectorDetail key={sectorIndex} data={sector} />;
```

Comment:
The tequila api-response provides an ID for each sector. I think it is time we add that id to the Sector object

Reply:
Are you sure? I think that there are ids for segment, but not sector.

Reply:
Oh, this might be what we discussed before. Yeah, I guess you are right.